openfisca_us/variables/gov/states/mo/tax/income/adjusted_gross_income/mo_taxable_income.py

```diff
@@ -0,0 +1,42 @@
+from openfisca_us.model_api import *
+
+
+class mo_taxable_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO taxable income"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.mo.gov/forms/MO-A_2021.pdf"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        threshold = parameters(period).gov.states.mo.tax.income.mo_taxable_income.mo_federal_itemization_threshold[filing_status]
+
+        #any itemizations 
+        agi = tax_unit("adjusted_gross_income", period)
+        total_itemized_federal_deductions = tax_unit("mo_federal_itemized_deductions", period)
+        if total_itemized_federal_deductions > 0:
```

Comment:
This needs to be a `where`

Since the logic is somewhat complex, you might want to break it up into a couple variables